src/richie/apps/courses/templates/courses/cms/fragment_course_content.html

```diff
@@ -85,15 +83,11 @@ <h2 class="section__title course-detail__content__row__title course-detail__cont
   <h2 class="section.title course-detail__content__row__title course-detail__content__team__title">
     {% trans 'Course team' %}
   </h2>
-  {% with header_level=3 %}
```

Comment:
Why do you remove this line? I think it is required to set the header here https://github.com/openfun/richie/blob/f960196fe378da814cfe4b0e9c27d25e35ecf4d5/src/richie/apps/courses/templates/courses/cms/fragment_person_glimpse.html#L25

Reply:
I didn't know it was required. I've added it back in.